string[] idle_robots
string[] assigned_jobs
string[] active_jobs

RobotStatus robot1
RobotStatus robot2
RobotStatus robot3

StationStatus station1
StationStatus station2
StationStatus station3
